packages/combobox/src/Combobox.tsx

```diff
@@ -47,7 +49,11 @@ import {
   menuWrapperStyle,
 } from './Combobox.styles';
 import { InternalComboboxGroup } from './ComboboxGroup';
-import { flattenChildren, getNameAndValue, OptionObject, keyMap } from './util';
+import {
+  flattenChildren,
+  getDisplayNameForValue,
+  getNameAndValue,
+} from './utils';
 
 /**
  * Component
```

Comment:
It would be cool if you could write something up for this before merging! :)

How's:
```
Combobox is a combination of a drop-down list or list box and a single-line editable textbox, allowing the user to either type a value directly or select a value from the list.
```